api/interservice/infra_proxy/request/policyfiles.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package chef.automate.domain.infra_proxy.request;
+option go_package = "github.com/chef/automate/api/interservice/infra_proxy/request";
+
+import "protoc-gen-swagger/options/annotations.proto";
+
+message Policyfiles {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+        json_schema: {
+          required: ["org_id,server_id"]
```

Comment:
This schema is the framework for beginning API documentation. If these fields in line 11 are not required, please edit!